src/apexcharts.js

```diff
@@ -706,15 +726,16 @@ export default class ApexCharts {
   revertDefaultAxisMinMax() {
     const w = this.w
 
-    w.config.xaxis.min = this.opts.xaxis.min || (Apex.xaxis && Apex.xaxis.min)
-    w.config.xaxis.max = this.opts.xaxis.max || (Apex.xaxis && Apex.xaxis.max)
+    w.config.xaxis.min = w.globals.lastXAxis.min
```

Comment:
@ThinkSalat This was old code and was reverted due to some issues. Can you please explain why this change is required?

Reply:
Oh, I didn't touch that. I might have pulled down an older release or something?

Reply:
Yes, that's a possibility.
Can you please pull the latest master branch and send a new PR?